x/farming/keeper/keeper.go

```diff
@@ -76,7 +76,7 @@ func (k Keeper) GetNextPlanIDWithUpdate(ctx sdk.Context) uint64 {
 	var id uint64
 	store := ctx.KVStore(k.storeKey)
 
-	bz := store.Get(types.GlobalPlanIdKey)
+	bz := store.Get(types.GlobalPlanIDKey)
```

Comment:
More discussion is needed, but why don't we unify it with `Id` to match the field name generated by proto?

Reply:
Ok.